examples/alexNet/setup.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
+wget --load-cookies /tmp/cookies.txt "https://docs.google.com/uc?export=download&confirm=$(wget --quiet --save-cookies /tmp/cookies.txt --keep-session-cookies --no-check-certificate 'https://docs.google.com/uc?export=download&id=1lH-GwVcNY0Jmf10zMRhGbbXaTe-LIH76' -O- | sed -rn 's/.*confirm=([0-9A-Za-z_]+).*/\1\n/p')&id=1lH-GwVcNY0Jmf10zMRhGbbXaTe-LIH76" -O alexNet.pt && rm -rf /tmp/cookies.txt
```

Comment:
Is it some official data or yours?

Reply:
If it is yours, I'm thinking of putting it here(https://github.com/hasktorch/libtorch-binary-for-ci/releases).

Reply:
Nah it's just the extracted parameters as mentioned in the [gist](https://gist.github.com/SurajK7/90de501ae7cf332b722b06a1dabb527d).

Any pointers to what went wrong with my GSoC proposal?

Reply:
There are some reasons for it, but I don't know if I can write it here.
Could you please check the feedback from @austinvhuang  in a private chat?

Reply:
BTW, thank you for sharing the gist.